charts/rasa-x/values.yaml

```
@@ -140,6 +140,17 @@ app:
   port: 5055
   # resources which app is required / allowed to use
   resources:
+  # envs is a dict containing app specific env variables
+  envs:
+  #  - name: DATABASE_URL
+  #    valueFrom:
+  #      secretKeyRef:
+  #        name: app-secret
+  #        key: database_url
+  # paths specifies the webhook and health check url paths of the action server app
+  paths:
```

For me personally, `paths` relates more to files. `endpoint` makes it a bit clearer that this API related in my opinion. However, would love to hear your opinion on this!
```suggestion
  # endpoints specifies the webhook and health check url paths of the action server app
  endpoints:
```

---

I tend to think of the service itself being an endpoint with one or more paths or "routes" within it.  I think however, it's a minor semantic difference and I'd be fine with calling it `endpoints` here as you've suggested.  I'll make that change as well.